src/test/java/calculator/PositiveNumber.java

```diff
@@ -0,0 +1,37 @@
+package calculator;
+
+import calculator.exception.NegativeNumberException;
+
+public class PositiveNumber {
+
+    private static final int MINIMUM_POSITIVE_NUMBER = 0;
+    private static final String EXCEPTION_MESSAGE_NEGATIVE_NUMBER = "문자열 계산기의 입력값은 음수가 될수 없습니다.";
+    private int number;
+
+    public PositiveNumber() {
+        this(0);
+    }
+
+    public PositiveNumber(String numberText) {
+        this(Integer.parseInt(numberText));
+    }
+
+    public PositiveNumber(int number) {
+        validation(number);
+        this.number = number;
+    }
+
+    private void validation(int number) {
+        if (number < MINIMUM_POSITIVE_NUMBER) {
+            throw new NegativeNumberException(EXCEPTION_MESSAGE_NEGATIVE_NUMBER);
+        }
+    }
+
+    public int getNumber() {
+        return number;
+    }
+
+    public void add(PositiveNumber positiveNumber) {
+        this.number += positiveNumber.number;
```

Comment:
`PositiveNumber`를 VALUE OBJECT로 관리하면 어떨까요?

```suggestion
        return new PositiveNumber(this.number + positiveNumber.number);
```